atomicapp/nulecule/__init__.py

```diff
@@ -0,0 +1,74 @@
+class NuleculeConfig(object):
+    pass
+
+
+class NuleculeBase(object):
+
+    def load(self):
+        pass
+
+    def run(self, provider):
+        raise NotImplementedError
+
+    def stop(self, provider):
+        raise NotImplementedError
+
+    def install(self):
+        raise NotImplementedError
+
+    def uninstall(self):
+        raise NotImplementedError
+
+
+class Nulecule(NuleculeBase):
+    """
+    This represents a Nulecule application. A Nulecule instance can have
```

Comment:
just this morning a little picky... "an Application compliant with the Nulecule Specification" ;)